source/_components/lifx.markdown

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: "LIFX"
+description: "Instructions on how to integrate LIFX into Home Assistant."
+date: 2016-01-27 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lifx.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.12
```

Comment:
Update this, since the component is new.

Reply:
That seems odd but okay. I am not sure what to put into `ha_category` for the component, though.